.github/actions/multistrap/action.yml

```diff
@@ -22,12 +26,20 @@ runs:
 
         multistrap_conf=""
         if [ "${{ inputs.arch }}" = "armv7" ]; then
-          multistrap_conf=multistrap_raspbian_buster.conf
           wget http://archive.raspbian.org/raspbian/pool/main/r/raspbian-archive-keyring/raspbian-archive-keyring_20120528.2_all.deb && sudo dpkg -i raspbian-archive-keyring_20120528.2_all.deb
         fi
-        if [ "${{ inputs.arch }}" = "aarch64" ]; then
+        if [ "${{ inputs.arch }}" = "armv7" ] && [ "${{ inputs.python-version }}" = "3.7" ]; then
+          multistrap_conf=multistrap_raspbian_buster.conf
+        fi
+        if [ "${{ inputs.arch }}" = "armv7" ] && [ "${{ inputs.python-version }}" = "3.9" ]; then
+          multistrap_conf=multistrap_raspbian_bullseye.conf
+        fi
+        if [ "${{ inputs.arch }}" = "aarch64" ] && [ "${{ inputs.python-version }}" = "3.7" ]; then
           multistrap_conf=multistrap_armbian64_buster.conf
         fi
+        if [ "${{ inputs.arch }}" = "aarch64" ] && [ "${{ inputs.python-version }}" = "3.9" ]; then
+          multistrap_conf=multistrap_armbian64_bullseye.conf
+        fi
```

Comment:
might be worth adding a fallback case that errors

Reply:
Fixed